packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -88,6 +91,75 @@ class MethodWebsocketRequestHandler {
     }
   }
 
+  MethodStreamManager _createMethodStreamManager(
+    WebSocket webSocket,
+    Server server,
+  ) {
+    return MethodStreamManager(
+      onInputStreamClosed: (
+        UuidValue namespace,
+        String parameterName,
+        CloseReason? closeReason,
+        MethodStreamCallContext callContext,
+      ) {
+        webSocket.tryAdd(
+          CloseMethodStreamCommand.buildMessage(
+            endpoint: callContext.endpoint.name,
+            method: callContext.method.name,
+            parameter: parameterName,
+            connectionId: namespace,
```

Comment:
Don't think so, as it's a concept only known to the stream manager. The call context isn't aware of the stream manager so I think this will obfuscate and make it harder to follow what's happening.